app/src/main/kotlin/com/wire/android/ui/home/conversations/MessageItem.kt

```diff
@@ -202,28 +220,45 @@ fun MessageItem(
                                 onLongClick = onLongClick,
                                 onOpenProfile = onOpenProfile
                             )
-                            MessageFooter(
-                                messageFooter,
-                                onReactionClicked
-                            )
                         } else {
                             MessageDecryptionFailure(
                                 messageHeader = header,
                                 decryptionStatus = header.messageStatus as MessageStatus.DecryptionFailure,
                                 onResetSessionClicked = onResetSessionClicked
                             )
                         }
+                        MessageFooter(
+                            messageFooter,
+                            onReactionClicked
+                        )
                     }
+                }
 
-                    if (message.sendingFailed) {
-                        MessageSendFailureWarning(header.messageStatus as MessageStatus.MessageSendFailureStatus)
-                    }
+                if (message.sendingFailed) {
+                    MessageSendFailureWarning(header.messageStatus as MessageStatus.MessageSendFailureStatus)
                 }
             }
         }
     }
 }
 
+@Composable
+fun EphemeralMessageExpiredLabel(conversationDetailsData: ConversationDetailsData) {
+    val stringResource = if (conversationDetailsData is ConversationDetailsData.OneOne) {
+        stringResource(
+            R.string.label_information_waiting_for_recipent_timer_to_expire_one_to_one,
+            conversationDetailsData.otherUserName ?: ""
```

Comment:
Couldn't we use a default string, something like: `Unknown` instead of the empty string?

Reply:
good point, I wonder if we will ever have situation when the other user name could be null, but thanks for pointing this out